### PR TITLE
login fix

### PR DESCRIPTION
**Pull request by wanghaoran1988**
### What type of PR is this?

_(bug/feature/cleanup/documentation)_
bug
### What this PR does / Why we need it?
* Fix proxy url is not write to kubeconfig correctly when using --proxy
* Update the backplane config structure
### Which Jira/Github issue(s) does this PR fix?

_Resolves #_

### Pre-checks (if applicable)

- [ ] Ran unit tests locally
- [ ] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
Config file looks like this:
```
cat ~/.config/.backplane.json 
{
    "url": "https://api.example.com",
    "proxy-url": "http://squid.proxy.example.com"
}

```

File: cmd/ocm-backplane/login/login.go [hunk 1]
```diff
@@ -170,9 +172,9 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	targetCluster.Server = bpAPIClusterUrl
 
 	// Add proxy URL to target cluster
-	globalOpts.ProxyURL = bpConfig.ProxyURL
```
Comment (samanthajayasinghe) [inline cmd/ocm-backplane/login/login.go:173]: I guess it's necessary to update proxy URL if it's not specified via `--proxy`
```
if globalOpts.ProxyURL == "" {
   globalOpts.ProxyURL = bpConfig.ProxyURL
}
```

Comment (supreeth7) [reply cmd/ocm-backplane/login/login.go:173]: It picks up from the config file if it isn't specified explicitly

Comment (samanthajayasinghe) [reply cmd/ocm-backplane/login/login.go:173]: nice :+1:

**Pull request merged**